src/databricks/labs/ucx/runtime.py

```diff
@@ -56,7 +55,6 @@ def all(cls):
                 ScanTablesInMounts(),
                 MigrateTablesInMounts(),
                 PermissionsMigrationAPI(),
-                ExperimentalWorkflowLinter(),
```

Comment:
don't remove this, keep it

Reply:
So we'll be linting jobs and dashboards twice ?

Reply:
we always need to keep a backup workflow for a release or two

Reply:
@nfx using 20mn, it times out... I don't have a solution (other than increasing the timeout to 25mns)